src/features/offer/components/VenueDetails/VenueDetails.tsx

```diff
@@ -8,6 +8,7 @@ import { getSpacing, Spacer, Typo } from 'ui/theme'
 import { getHoverStyle } from 'ui/theme/getHoverStyle/getHoverStyle'
 
 export interface VenueDetailsProps extends VenueDetail, ViewProps {
+  // only used by venue card
```

Comment:
```suggestion
  /**
    * This prop is for internal usage only.
    * It is used by `<VenueCard />` component and should not be used by
    * developers outside this component.
    */
```

Pour que ce soit bien clair 😛